src/createScaffoldMiddleware.ts

```diff
@@ -19,7 +19,7 @@ type ScaffoldMiddlewareHandler<
 export function createScaffoldMiddleware(handlers: {
   [methodName: string]: ScaffoldMiddlewareHandler<JsonRpcParams, Json>;
 }): JsonRpcMiddleware<JsonRpcParams, Json> {
-  return (req, res, next, end) => {
+  return async (req, res, next, end) => {
```

Comment:
The linter is asking for this, because middleware can now be async. We could create sync and async middleware types, not sure if it's worth it.